Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -2385,7 +2385,10 @@
     (deny mach-lookup (with telemetry-backtrace)
         (require-all
             (require-not (extension "com.apple.webkit.extension.mach"))
-            (xpc-service-name "com.apple.MTLCompilerService"))))
+            (xpc-service-name "com.apple.MTLCompilerService")))
+    (deny iokit-open-service (with no-report)
+        (iokit-registry-entry-class
+            "AMDRadeonX6000_AMDNavi14GraphicsAccelerator"))) ;; <rdar://104970117>
```

Comment:
Perhaps we could add no-report to the existing deny iokit-open-service rule, since all IOKit should be blocked?

Reply:
Can you have "no-report" and "telemetry-backtrace" in the same rule? And do we want to keep the backtrace if we are silencing?

I could change the rule to just be `(with no-report)` if you are okay with that.

I'd prefer to generate telemetry if we don't think there are cases where this will happen anymore.

Reply:
> Can you have "no-report" and "telemetry-backtrace" in the same rule? And do we want to keep the backtrace if we are silencing?

Yes, I believe you can have that, since no-report is only affecting local reporting.

> 
> I could change the rule to just be `(with no-report)` if you are okay with that.
> 
> I'd prefer to generate telemetry if we don't think there are cases where this will happen anymore.

I also think it would be good to keep telemetry a bit longer, since these rules have just been enabled.